.gitignore

```diff
@@ -0,0 +1,10 @@
+# IntelliJ project files
```

Comment:
These ignores should be added to your personal ignore files (either the global one (in case you have created one), or `.git/info/exclude`), not to the project's `.gitignore`.

Reply:
Already have them in my `~/.gitignore_global` ; ). Just added these lines, as too many "Newbies" push those files by accident - we might even add more commonly used editors like PHPStorm or suchlike, don't you think?

Reply:
Adding these ignores to `.gitignore` could prevent future users and contributors from accidentally committing their IDE's helper files, but the problem with adding ignores for specific IDEs is that (in my opinion) this also means we should add ignores for most of the other popular IDEs, which would bloat the project's `.gitignore` and make it hard to maintain. I personally think that `.gitignore` files should always be as simple and short as possible, since anyone can customize their local ignore files in case they'd like to.

We could add a note about this to the readme.

Reply:
#done - removed from local .gitignore & added information to README